app/uk/gov/hmrc/taxhistory/metrics/MetricsEnum.scala

```diff
@@ -25,4 +25,5 @@ object MetricsEnum extends Enumeration {
   val NPS_GET_IABDS = Value
   val NPS_GET_TAX_ACCOUNT = Value
   val CITIZEN_DETAILS = Value
+  val DES_NPS_GET_EMPLOYMENTS = Value
```

Comment:
We've already got `NPS_GET_EMPLOYMENTS` - is this the same thing?

Reply:
This is to differentiate between squid and des calls for get employments. If not needed then use the `NPS_GET_EMPLOYMENTS`

Reply:
These metrics end up in the grafana dashboard (https://grafana.tools.production.tax.service.gov.uk/d/LvOXXDOmz/tax-history?orgId=1) as just "NPS Get Employment Call Count" and "NPS Get Employment Response Time".
If we use a different metric name, we'd need to change the Grafana dashboard configuration too. 
We wont need to distinguish between Squid and DES in the metrics though, so we may as well keep the same previous name of `NPS_GET_EMPLOYMENTS`.